mate-panel/libmate-panel-applet-private/panel-applet-frame-dbus.c

```diff
@@ -235,33 +235,32 @@ container_child_background_set (GObject      *source_object,
 
 	mate_panel_applet_container_child_set_finish (container, res, NULL);
 
-	if (frame->priv->bg_cancellable)
-		g_object_unref (frame->priv->bg_cancellable);
-
-	frame->priv->bg_cancellable = NULL;
+	if (frame->priv->bg_operation)
+		frame->priv->bg_operation = NULL;
```

Comment:
This seems a bit off, should that be if(frame->priv->bg_cancellable) ?
Same thing a few lines down.

Reply:
Na, it should not. But the if should be removed I guess

Reply:
Yeah, `bg_cancellable` is gone. Indeed, I can just assign NULL to `bg_operation`, should be the same.